inginious/frontend/pages/register.py

```diff
@@ -201,14 +205,19 @@ def POST(self):
         msg = ""
         error = False
         data = flask.request.form
+        loop = flask.current_app.motor_loop
         if "register" in data:
-            msg, error = self.register_user(data)
+            future = asyncio.run_coroutine_threadsafe(self.register_user(data), loop)
```

Comment:
We should transform GET and POST methods into coroutine. Flask seems to embed some support for asyncio : https://flask.palletsprojects.com/en/stable/async-await/

We should probably just find out how to use this feature with the Flask `MethodView` class